packages/aws-cdk-lib/aws-logs/lib/log-group.ts

```diff
@@ -245,6 +267,64 @@ abstract class LogGroupBase extends Resource implements ILogGroup {
 
     return principal;
   }
+
+  /**
+   * Creates a CloudWatch metric for the number of incoming log events to this log group.
+   *
+   * @param props - Optional. Configuration options for the metric.
+   * @returns A CloudWatch Metric object representing the IncomingLogEvents metric.
+   *
+   * This method allows you to monitor the rate at which log events are being ingested
+   * into the log group. It's useful for understanding the volume of logging activity
+   * and can help in capacity planning or detecting unusual spikes in logging.
+   *
+   * Example usage:
+   * ```
+   * const logGroup = new logs.LogGroup(this, 'MyLogGroup');
+   * const incomingEventsMetric = logGroup.metricIncomingLogEvents();
+   * new cloudwatch.Alarm(this, 'HighLogVolumeAlarm', {
+   *   metric: incomingEventsMetric,
+   *   threshold: 1000,
+   *   evaluationPeriods: 1,
+   * });
+   * ```
+   */
+  public metricIncomingLogEvents(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return this.metric('IncomingLogs', props);
```

Comment:
According to the documentation, the only valid statistic is `Sum` so we should set the statistic to sum.

Reply:
Should have addressed all the comments above.